pandas/tests/arrays/datetimes/test_constructors.py

```diff
@@ -242,6 +243,17 @@ def test_from_arrowtest_from_arrow_with_different_units_and_timezones_with_(
     tm.assert_extension_array_equal(result, expected)
 
 
+def test_datetimetz_from_arrow_roundtrip():
```

Comment:
Is this additional test needed if the above is corrected?

Reply:
Nope, updated.